test/integration/index-test.js

```diff
@@ -365,7 +365,8 @@ describe(function() {
         fixturesPath: 'test/fixtures/app/non-default-addon-blueprint/local',
         commitMessage: 'my-app',
         packageName: location,
-        to,
+        from,
```

Comment:
Why change this?

Reply:
I believe `loadSafeBlueprintFile(fixturePath)).blueprints` is an array of objects whose `version` property is the one set in the config that the project currently has installed. That to me sounds like the version we are going `from` . The test fails if I don't have this change

Reply:
I think when you moved this test, you got the fixture wrong. It was working.

Reply:
Ohh I see what happened.